### DISCUSSION TITLE
请问 已验证地址列表(可通过 cf 内部 api 发送邮件)具体指什么，不是很理解

### DISCUSSION THREAD

**Discussion by 80imike** [dreamhunter2333/cloudflare_temp_email#299] [Q&A] [answered]
这里的已验证地址列表指的是在哪里验证的？不是很理解

<img width="862" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/04d294ac-4702-41bd-9aec-524f4f76b6b6">

**Comment by dreamhunter2333** [comment]
> 
> 
> 这里的已验证地址列表指的是在哪里验证的？不是很理解
> 
> 
> 
> <img width="862" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/04d294ac-4702-41bd-9aec-524f4f76b6b6">
> 
> 

cloudflare 邮件路由中认证的邮件

**Comment by 80imike** [comment]
> > 这里的已验证地址列表指的是在哪里验证的？不是很理解
> > <img alt="image" width="862" src="https://private-user-images.githubusercontent.com/1665201/334458492-04d294ac-4702-41bd-9aec-524f4f76b6b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5NDcwODIsIm5iZiI6MTcxNjk0Njc4MiwicGF0aCI6Ii8xNjY1MjAxLzMzNDQ1ODQ5Mi0wNGQyOTRhYy00NzAyLTQxYmQtOWFlYy01MjRmNGY3NmI2YjYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMDEzOTQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2FkMGNhNzI3MWRlZWE4YWU2M2NmZGM2Y2E2NzkyYzU1ODMxMDAyYzU1MzcxNDJjNTRkODZiYTU1YTcwNWY1NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.PgBNM0LZP3H62INui7QXQ-F-JScXeISNmKhNpo6E3Pw">
> 
> cloudflare 邮件路由中认证的邮件

![image](https://github.com/dreamhunter2333/cloudflare_temp_email/assets/1665201/866ec0dc-ff01-453c-b512-23dd8844b308)


是指这里加吗？ 我加了但发送邮件时报错 Code 400: Failed to send mail Mailchannels error: 500 {"errors":["Failed to send email: 550 5.7.1 This sender is not authorized to send from xxxx.top 


@dreamhunter2333 


----

问题已解决，大佬不用看了

**Reply by lovewe** [reply]
大佬，我也不懂，请问这个下拉列表怎么才能出现数据？

**Reply by dreamhunter2333** [reply, marked as answer]
> 大佬，我也不懂，请问这个下拉列表怎么才能出现数据？

这个是手动输入的

**Comment by 3377** [comment]
请教一下，怎么解决的。这个文档写的真的是一言难尽，写的只有作者自己懂。

**Comment by 3377** [comment]
研究了2个小时，最后一个下拉框。是手动输入的。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。
厉害厉害厉害。
